sunpy/time/time.py

```diff
@@ -124,88 +142,73 @@ def _astropy_time(time):
     return time if isinstance(time, astropy.time.Time) else astropy.time.Time(parse_time(time))
 
 
-def _parse_dt64(dt):
-    """
-    Parse a single numpy datetime64 object
-    """
-    # Convert to microsecond precision because datetime cannot handle nanoseconds
-    return dt.astype('M8[us]').astype(datetime)
-
-
 @singledispatch
-def convert_time(time_string, **kwargs):
+def convert_time(time_string, format=None, **kwargs):
     # default case when no type matches
-    raise ValueError("'{tstr!s}' is not a valid time string!".format(tstr=time_string))
+    return Time(time_string, format=format, **kwargs)
 
 
 # Only register pandas if we can import pandas
 try:
     import pandas
 
-    @convert_time.register(pandas.Timestamp)
```

Comment:
This shouldn't be removed. Something happened while merging in master.
Replace with 
```python
    @convert_time.register(pandas.Timestamp)
    def convert_time_pandasTimestamp(time_string, **kwargs):
        return time_string.to_pydatetime()
```

Reply:
I think in the merged I added it twice, so I removed one. But I will check to make sure.

Reply:
Added it back.